tests/static_map/shared_memory_test.cu

```diff
@@ -123,7 +123,9 @@ TEMPLATE_TEST_CASE_SIG("Shared memory static map",
     }
     thrust::device_vector<ref_type> d_refs(h_refs);
 
-    auto constexpr num_windows = cuco::experimental::make_window_extent<ref_type>(extent_type{});
+    // maybe_unused to silence false positive "variable set but not used" warning
+    [[maybe_unused]] auto constexpr num_windows =
```

Comment:
This workaround is fine. Should we get the compiler team involved so they can investigate?

Reply:
Based on offline discussions, it's been a known gcc issue for a while.